scripts/package.json

```diff
@@ -41,5 +41,8 @@
     "tslint": "^5.8.0",
     "typescript": "^2.5.3",
     "webpack": "^3.8.1"
+  },
```

Comment:
what's `define-property` being used for?

Reply:
I don't know, it was that weird thing you had me changed so that grunt build would work

Reply:
👍